src/scene/loading.cpp

```diff
@@ -120,7 +120,7 @@ namespace ace { namespace scene {
         }
 
         this->frame.progress_bar->value = client.net.map_writer.vec.size();
-        this->frame.progress_bar->range = std::max(1u, client.net.map_writer.vec.capacity());
+        this->frame.progress_bar->range = std::max(1u, (uint)client.net.map_writer.vec.capacity());
```

Comment:
grr I hate unsigned types. Just replace `1u` with `size_t(1)` and revert the `(uint)` cast.